### DISCUSSION TITLE
質問における明らかなオフトピおよびマルチポスト指摘に対する対処について

### DISCUSSION THREAD

**Discussion by Verclene** [increments/qiita-discussions#342] [ヘルプ] [answered]
Qiita運営,コミュニティの皆様

日頃から運営お疲れ様です.

特定個人名を出すことは控えますが,
最近Q&Aページにおいて,質問文の情報不足を差し引いても明らかに見当違い,あるいは問題解決という観点に即してまったく役に立たないと思われる回答が散見され困惑しております.
中には全く内容に関係のないマルチポストの指摘だけしていく人もいて,無回答の質問が蓄積するのはまだしも,こうした意味のない回答だけがついている質問がWeb検索で引っかかる状態はいかがなものかと感じます.(なお,Qiitaにおいてマルチポストを禁止する事項は利用規約には存在しないとの認識です.齟齬がありましたらご指摘くだされば幸いです)

Stackoverflowのdownvoteのような機能や,運営としての対処方針等,ご意見をお聞かせくださればと存じます.

**Comment by tanaka-asami415** [comment, marked as answer]
@Verclene 
ご報告ありがとうございます🙏

利用規約に違反していなくても、コミュニティガイドライン違反に該当する場合には
個別に対応したいと思っておりますので、[通報機能](https://help.qiita.com/ja/articles/qiita-report)よりご報告いただけますと幸いです。
違反しているかの判断が難しい場合には、[サポートフォーム](https://support.qiita.com/hc/ja/requests/new)へ該当のページをご連絡ください。

downvoteのような機能については度々社内で議論はしているのですが、現時点では実施の予定はございません🙇

**Reply by Verclene** [reply]
その後直近でも何件か通報させていただいておりますが,改善の傾向がが見られません.
特定ユーザの回答をみているのか,荒らしのような状態です.
運営として頓珍漢かつ悪質な回答に対する対応はしないおつもりなのでしょうか.

**Reply by tanaka-asami415** [reply]
@Verclene 
明らかな規約違反行為が認められなかった場合には、1ユーザーさまからの通報だけでなく、
複数アカウントからの通報等も合わせての総合判断となります🙇

また、強制的な対応をしない場合でも、コミュニティガイドラインに沿って
運営からユーザー様への警告等の対応を順次行っています。

**Comment by nak435** [comment]
初めまして。Qiitaにはいつもお世話になっています。
今までdiscussionsの存在を知らず、今回、類似の内容がないか見てみて、初めてこちらに書くことにしました。

内容は、
「質問」される方が、その質問のためだけにアカウントを作成していると思われるような方への対応です。
それだけなら、「まあそういうこともあるのかな」なんですが、
回答者へのお礼の一言もなく、クローズもしないで放置しているものが散見され、
マナー的にいかがなものかと思います。
そういう方へは、回答する気も失せるので、「質問」に特化した何等かガイドライを示すのが良いのではないでしょうか。
（「回答者への配慮」とか「解決ならクローズする」とかを明文化する）
回答が寄せられた後に、一定期間経過したものを、自動的にクローズする強行対応もありかと思います。

ご検討いただけると幸いです。
もし見当違いな指摘であるなら、ご容赦ください。

**Reply by Verclene** [reply]
当初の議題は回答者側の行為を問題視するもので,質問したっきりみたいな問題はあんまり関係ないと思われるので新規にスレッドを立てることを提案しておきます.
件の回答の傾向や現状を見る限り,何らかの方針を議論する方はいないものとみて一旦閉めます.

**Reply by nak435** [reply]
> 新規にスレッドを立てることを提案しておきます.

失礼しました。検討いたします。

**Reply by tanaka-asami415** [reply]
@nak435 
初めてのコメントありがとうございます🙏

> 「質問」に特化した何等かガイドライを示すのが良いのではないでしょうか。

質問の編集画面に「良い質問を書くには」タブから[良い質問を書くためのガイドライン](https://help.qiita.com/ja/articles/qiita-question-guideline) を
確認することができるのですが、それでは不十分であると理解しております。

「良い質問を書くためのガイドライン」の内容の更新と
視認性については、質問ではないのですが他にも同様のご意見をいただいているため https://github.com/increments/qiita-discussions/discussions/231

今後の課題として合わせて検討させていただきます🙇

@Verclene 
> 何らかの方針を議論する方はいないものとみて一旦閉めます.

承知いたしました。こちらのDiscussionはクローズさせていただきます。
